src/main/resources/templates/taskform.html

```diff
@@ -58,6 +58,11 @@
                     <input type="submit" value="Save"/>
                 </div>
             </div>
+            <div class="task-field">
+                <div class="button">
+                    <input type="button" value="Cancel"/>
```

Comment:
Don't add this, until it is fully functional.  You shouldn't merge incomplete features into master.

Reply:
Thanks for the tip. 
Its a good thought, I'll add to this feature and then go ahead and merge it in.